File: programacao/c/respostas/operacoes/14.c
#include <stdio.h>

int main() {
    const float PI = 3.14159;
    float raio, altura, volume;

    printf("Digite o raio da base do cilindro (cm): ");
    scanf("%f", &raio);

    printf("Digite a altura do cilindro (cm): ");
    scanf("%f", &altura);

    volume = PI * raio * raio * altura;

    printf("A capacidade da lata cilindrica e %.2f cm^3\n", volume);

    return 0;
}

File: programacao/python/respostas/operacoes/14.py
import math

raio = float(input("Raio da base do cilindro (cm): "))
altura = float(input("Altura do cilindro (cm): "))
volume = math.pi * raio * raio * altura
print(f"A capacidade da lata cilindrica e {volume:.2f} cm^3")
